src/contentParser.js

```diff
@@ -31,6 +31,23 @@ export default function contentParser(
     return content;
   }
 
+  const subdirectoryCorrection = path => {
+    const wordPressUrlParsed = new URIParser(wordPressUrl);
+    // detect if WordPress is installed in subdirectory
+    const subdirectoryRegex = new RegExp(
```

Comment:
wordPressUrlParsed.path() should be the subdir (or /)

Reply:
also please pass wordPressUrl as a function parameter

Reply:
Sure, easier and shorter. I also pass wordPressUrl as param

Reply:
I've committed the refactored version